<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="red">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border: none;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="blue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;
border: none;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="green">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;
border: none;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: #fff;
border: none;</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="up">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="down">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selector">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>51</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="power">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="screen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>501</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="menuLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>300</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mro</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QListWidget" name="menu">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>60</y>
       <width>461</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="heart">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="achievementLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Achievement</string>
     </property>
    </widget>
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>271</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="coherenceLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Coherence</string>
     </property>
    </widget>
    <widget class="QLabel" name="lengthLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QLabel" name="length">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coherence">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="achievement">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="batteryBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="breathPacer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QGraphicsView" name="beep">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: yellow;
border: none;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="hrContactLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans Multani</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>HR Contact:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hrContact">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="charge">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>screen</zorder>
   <zorder>red</zorder>
   <zorder>blue</zorder>
   <zorder>green</zorder>
   <zorder>menuButton</zorder>
   <zorder>returnB</zorder>
   <zorder>up</zorder>
   <zorder>down</zorder>
   <zorder>left</zorder>
   <zorder>right</zorder>
   <zorder>selector</zorder>
   <zorder>power</zorder>
   <zorder>hrContactLabel</zorder>
   <zorder>hrContact</zorder>
   <zorder>charge</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
